fs/fs_zenfs.cc

```diff
@@ -1171,6 +1172,18 @@ std::map<std::string, std::string> ListZenFileSystems() {
   return zenFileSystems;
 }
 
+void ZenFS::GetZoneSnapshot(std::vector<ZoneSnapshot>& zones) {
+  zbd_->GetZonesSnapshot(zones);
+}
+
+void ZenFS::GetZoneFileSnapshot(std::vector<ZoneFileSnapshot>& zone_files) {
+  files_mtx_.lock();
```

Comment:
For locking and unlocking `files_mtx_` we use this pattern:
`std::lock_guard<std::mutex> file_lock(files_mtx_);`

Reply:
Fixed.